reports/models.py

```diff
@@ -190,3 +191,33 @@ class UploadLevelTotals(AbstractTotals):
 
     class Meta:
         db_table = "reports_uploadleveltotals"
+
+
+class Test(BaseCodecovModel):
+    name = models.TextField()
+    repository = models.ForeignKey(
+        "core.Repository", related_name="tests", on_delete=models.CASCADE
+    )
+
+    class Meta:
+        db_table = "reports_tests"
+
+
+class TestSession(BaseCodecovModel):
+    report_session = models.OneToOneField(ReportSession, on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "reports_testsessions"
+
+
+class TestRun(BaseCodecovModel):
+    outcome = models.BooleanField()
+    duration = models.FloatField()  # number of seconds
```

Comment:
if you call the field `duration_seconds` you don't need the comment and it's easier to know that it's seconds when using the field